app/views/settings.js

```diff
@@ -25,6 +25,7 @@
                 allowBugReporting: !(await F.state.get("disableBugReporting")),
                 allowUsageReporting: !(await F.state.get("disableUsageReporting")),
                 hasPushNotifications: !!(await F.state.get('serverGcmHash')),
+                isElectron: (F.electron),
```

Comment:
The parans are a noop, but what you might want (but not strictly required) is a `!!F.electron` which coerces the F.electron truthiness to a boolean.